shell/browser/api/atom_api_app.cc

```diff
@@ -850,7 +850,7 @@ void App::SetAppLogsPath(mate::Arguments* args) {
   base::FilePath custom_path;
   if (args->GetNext(&custom_path)) {
     if (!custom_path.IsAbsolute()) {
-      args->ThrowError("Path must be absolute");
+      args->ThrowError("Path must be absolute.");
```

Comment:
```suggestion
      args->ThrowError("Path provided to setAppLogsPath must be absolute.");
```

If that is the name of the method?